app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -68,6 +71,36 @@ protected function create(array $data)
             'last_name' => $data['last_name'],
             'email' => $data['email'],
             'password' => bcrypt($data['password']),
+            'email_token' => base64_encode($data['email'])
```

Comment:
This is a security flaw. If the algorithm is exposed it's very easy to create tokens to activate other user's accounts. https://3v4l.org/kGcO0 shows you how this works. It's always the same result. So I could activate your account by just adding the base64 encoded value to the link.

Better to use `str_random` to have a random string which cannot be guessed.